### ISSUE TITLE
CMakeLists.txt requires only CMake 3.16 but project options use policy from 3.18

### ISSUE THREAD

**Issue by dholmes215** [cpp-best-practices/cmake_conan_boilerplate_template#17]
From this discussion on Discord: https://discord.com/channels/773320866993012746/954385320336556042/958776874895556698

A user encountered the error:

```cmake
CMake Error at build/_deps/_project_options-src/src/PackageProject.cmake:4 (cmake_policy):
  Policy "CMP0103" is not known to this version of CMake.
```

That policy was added in CMake 3.18, but this project's main CMakeLists.txt uses `cmake_minimum_required(VERSION 3.16)`.

**Comment by lefticus** [comment]
@ddalcino very good point. I'll create a new branch that we can collaborate on to get the docs/dependencies in alignment, and test upgrading to project_options 0.20.0

We can also use that branch to discuss if we want our own fork of project_options.

**Comment by lefticus** [comment]
See #18 

Looks like we have work to do. Seems conan is no longer running. Somehow was disabled in the dynamic_project_options defaults, perhaps?

**Comment by ddalcino** [comment]
Looks like it happened here: https://github.com/aminya/project_options/pull/71

Should be easy enough to specify `ENABLE_CONAN` when we call `dynamic_project_options()` though.

**Comment by lefticus** [comment]
Fixed in e51378d6 I think
